### PR TITLE
 Initial work to get cloud.Topics working on azure servicebus. 

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Note: i have not been able to test this yet due to the azure outage currently in effect.  I'm just sending this out for some preliminary eyes.

### CODE REVIEW

File: azure/bucket.ts [hunk 1]
```diff
@@ -104,13 +104,6 @@ export class Bucket extends pulumi.ComponentResource implements cloud.Bucket {
             filter = filter || {};
             serverless.storage.onBlobEvent(putName, storageAccount, {
                     func: (context, buffer) => {
-                        // Redirect console logging to context logging.
```
Comment (swgillespie) [inline azure/bucket.ts]: Why'd this get removed?

File: azure/topic.ts [hunk 2]
```diff
@@ -13,22 +13,67 @@
...
+            location: shared.location,
+            resourceGroupName: shared.globalResourceGroupName,
+            // topics are only supported in standard and premium.
+            sku: "standard",
+        }, { parent: this });
+
+        const topic = new azure.eventhub.Topic(name, {
+            resourceGroupName: shared.globalResourceGroupName,
+            namespaceName: namespace.name,
+        });
```
Comment (swgillespie) [inline azure/topic.ts]: need a parent for this?

File: azure/bucket.ts (hunk 1 above)
Comment (CyrusNajmabadi) [reply azure/bucket.ts]: It got moved down to azure-serverless.

File: azure/topic.ts (hunk 2 above)
Comment (CyrusNajmabadi) [reply azure/topic.ts]: yes!

**Pull request merged**